core/src/test/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderIntegrationTest.kt

```diff
@@ -62,6 +63,7 @@ import org.mockito.kotlin.spy
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.verifyNoInteractions
 import org.mockito.kotlin.whenever
+import org.testcontainers.shaded.org.bouncycastle.util.encoders.Hex
```

Comment:
issue: I am not sure if this is what you wanted to use. I believe you should use `io.netty.buffer.ByteBufUtil.decodeHexDump` instead